bun.lock

```diff
@@ -14,7 +14,7 @@
     },
     "example": {
       "name": "example",
-      "version": "0.1.14",
+      "version": "0.1.17",
```

Comment:
Why are we bumping this manually (and by 3 versions at once)? Shouldn't it be bumped automatically by the release job?

Reply:
it's the example app and it bumped after running `bun install`